.travis.yml

```diff
@@ -17,12 +17,10 @@ script:
 
 jobs:
   include:
-      # Test Stage
+    # Test Stage
     - stage: test
       python: 2.7
       script: pytest --cov pylintfileheader pylintfileheadertest # Do not lint here
-    - stage: test

```

Comment:
I would prefer to keep the python 3.4 CI build. Do you have an idea how we could retain the python 3.4 build?

Reply:
working on it

Reply:
Done by pinning the `readme-renderer` to `24.0` for python3.4